core/drivers/crypto/caam/caam_ctrl.c

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/**
+ * @copyright 2017-2019 NXP
+ *
+ * @file    caam_ctrl.c
+ *
+ * @brief   CAAM Global Controller.\n
+ */
+
+/* Global includes */
+#include <initcall.h>
+#include <tee_api_types.h>
+
+#ifdef CFG_CRYPTO_DRIVER
+/* Driver Crypto includes */
+#include <drvcrypt.h>
+#endif
+
+/* Local includes */
+#include "caam_common.h"
```

Comment:
@jenswi-linaro 
Agree with you in other implementation when the include files are in global include directory, but in this case, the `core/drivers/crypto/caam/include` are kept locally to the `core/driver/crypto/caam `sources. 

I didn't want to make those include accessible from other sources as they are used only in the caam driver implementation. The path is not global-incdir in the sub.mk, this is why I kept the local include definition. There is nothing exported and to be re-used by other part of code elsewhere. Files are completely private to the local driver implementation.

Now if you prefer me to moved them in the `core/include/drivers/crypto/caam`, I'll do it and so prefix all include file name with `caam_`. 

Please let me know if you still want me to move file in global include directory.

Reply:
I understand, no need to move the .h files to a global include directory.

However, the files included via the include paths (using `-I` argument to the compiler) should still preferably be included with `#include <file.h>` rather than `#include "file.h"`. All files included via include paths can cause conflicts so prefixing is still needed, unless they are included as `#include <caam/file.h>`, then `caam/` is the prefix.

Reply:
ok understood. I'll add file prefix and change the `#include"file.h"` to `#include <file.h>`